### DISCUSSION TITLE
Support navigation keys on full keyboard (eg. home, end, page up/down)

### DISCUSSION THREAD

**Discussion by gkertai** [videovillage/community#106] [Ideas]
When playing back videos, I have a habit of hitting ↖HOME on my keyboard to wind it back to the beginning, but Screen does not respond to this keystroke. It does not handle the ↘END key either for jumping to the end. I searched the help menu, but there seem to be no functions to support these. 

How big of a hassle would it be to add them to a next version?

The best workaround I have is using ⇧I and ⇧O to jump to the beginning/end, but it's not nearly as convenient.

On a similar note, I would also love to see the ⇞ Page Up and ⇟ Page Down keys utilized for jumping between markers. They're quicker to hit than ⇧↑ and ⇧↓ respectively.

Thanks and best regards!

**Comment by gregcotten** [comment]
Oh neat! These should be easy to add.
